src/main/java/com/virtuslab/using_directives/config/Settings.java

```diff
@@ -0,0 +1,30 @@
+package com.virtuslab.using_directives.config;
+
+public class Settings {
+    private boolean allowRequire = true;
```

Comment:
Would having a list of allowed 'using keywords' work? This would be more generic and allow for the same functionality of allowing/disallowing using require.

Reply:
It might be hard because each keyword is a new token which would need to be added dynamically and I think that some keywords might not work.
e.g if somebody adds keyword `class`, in Scala file we can try to parse class definition as using directive